cogboard-webapp/src/components/widgets/index.js

```diff
@@ -177,3 +177,9 @@ const widgetTypes = {
 };
 
 export default widgetTypes;
+
+export function getWidgetType(widgetType) {
+  return widgetTypes.hasOwnProperty(widgetType)
+    ? widgetTypes[widgetType]
+    : widgetTypes['WhiteSpaceWidget'];
+}
```

Comment:
Please move this function outside `/widgets/index.js` (for example `helpers.js`). This file is used only for registering new widgets.

Reply:
Fixed